src/app/shop/[category]/_components/TitleWrap/TitleWrap.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import CategoryTitle from '@/app/shop/_components/Category/CategoryTitle';
+import { CATEGORY_MAP } from '@/constants/product';
+import { CategoryKey } from '@/types/Category';
+import classNames from 'classnames/bind';
+import { useState } from 'react';
+import Filter from './Filter';
+import FilterSort from './FilterSort';
+import styles from './TitleWrap.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface TitleWrapProp {
+  category: CategoryKey;
+  totalCount: number;
+}
+export default function TitleWrap({ category, totalCount }: TitleWrapProp) {
+  const [isOpen, setIsOpen] = useState(false);
+  const handleClickFilter = () => setIsOpen(!isOpen);
```

Comment:
괄호로 감싸는 게 좋을 것 같고, prevState 이용하는 게 더 안전할 것 같습니다!
```suggestion
  const handleClickFilter = () => {
    setIsOpen((prev) => !prev);
  }
```

Reply:
오 감사합니당!!